client/service/clusters.go

```diff
@@ -236,10 +236,15 @@ func (a ClustersAPI) GetOrCreateRunningCluster(name string, custom ...model.Clus
 			return cl, nil
 		}
 	}
-	instanceType := "m4.large"
-	if a.client.IsAzure() {
-		instanceType = "Standard_DS3_v2"
+
+	nodeTypes, err := a.ListNodeTypes()
+	if err != nil {
+		return
 	}
+
+	instanceType := nodeTypes[0].NodeTypeID
```

Comment:
one cannot pick just any node type - it must be the one without the need to attach block storage & cheap :)

Reply:
my advice will be: query node types, sort all nodes, pick
```
{
      "memory_mb": <= lowest memory
      "num_cores": <= lowest cores
      "description": "m5a.4xlarge (beta)",
      "instance_type_id": "m5a.4xlarge",
      "is_deprecated": <= non-deprecated
      "category": "General Purpose",
...
      "num_gpus": 0,
      "node_instance_type": {
        "local_disks":                  <= with a local disk
        "local_disk_size_gb": 0
      },
    },
```

Reply:
Alright, sorted by deprecation, memory and CPU. But I don't see how to sort by the other parameters you mentioned.

Reply:
Found it, they were just not mapped yet, on it